apps/frontend-pwa/src/pages/session/[id].tsx

```diff
@@ -1,27 +1,95 @@
-import { useQuery } from '@apollo/client'
+// TODO: remove solution data in a more specialized query than getSession (only the active instances are required)
+// TODO: move layout around questions and feedback to separte component for reusability
+
 import { GetSessionDocument } from '@klicker-uzh/graphql/dist/ops'
+import { GetServerSideProps } from 'next'
+// import { useQuery } from '@apollo/client'
+
+import FeedbackArea from '@components/liveSession/FeedbackArea'
+import QuestionArea from '@components/liveSession/QuestionArea'
+import { initializeApollo } from '@lib/apollo'
+import Head from 'next/head'
 import { useRouter } from 'next/router'
+import { twMerge } from 'tailwind-merge'
 
-function Index() {
+function Index({ session }: any) {
```

Comment:
use the type that is returned from the query in server side props. otherwise, we lose all of the typings here